pkg/monitoring/vmistats/collector_test.go

```diff
@@ -108,12 +108,12 @@ var _ = Describe("Utility functions", func() {
 
 			countMap = makeVMICountMetricMap(nil)
 			Expect(countMap).NotTo(BeNil())
-			Expect(len(countMap)).To(Equal(0))
+			Expect(countMap).To(HaveLen(0))
```

Comment:
I guess you probably wanted to use `BeEmpty()` here as well.

Reply:
`countMap` is a `map`, not `unit64`, so using `BeEmpty` makes sense.

Reply:
Ah, you're right. Thank you.

Reply:
Done.